controllers/whoToFollow/whoToFollowV2.js

```diff
@@ -81,6 +82,8 @@ module.exports = async (req, res) => {
             ...userTopic,
             viewtype: 'user'
           });
+
+          allUserIds.push(userTopic.user_id);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

While pushing `user_id` into the `allUserIds` array, it would be a good idea to check if the `user_id` already exists in the array. This will prevent duplicate entries and keep the array size minimal, improving performance.
```diff
+           if (!allUserIds.includes(userTopic.user_id)) {
               allUserIds.push(userTopic.user_id);
+           }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->